playbook/hosts

```diff
@@ -4,8 +4,16 @@ guide.aelve.com
 s1.codesearch.aelve.com
 s1.staging.codesearch.aelve.com
 
+[all:vars]
+ansible_python_interpreter=/usr/bin/python3
+guide_main_port=8080
+guide_api_port=4400
+guide_ekg_port=5050
+image_name=develop--back
+
 [terminator]
 terminator.aelve.com
 
-[all:vars]
-ansible_python_interpreter=/usr/bin/python3
+[staging-guide]
```

Comment:
Would be better to name it "guide-staging", because in the future we might have "guide-staging, guide-prod, guide-whatever"